README.md

```diff
@@ -350,6 +350,39 @@ foreach ($lookahead as $value) {
 The `Hoa\Iterator\Lookbehind` also exists and allows to look behind for the
 previous element.
 
+### Buffer
+
+`Hoa\Iterator\Buffer` allows to move forward as usual but also backward up to a
+given buffer size over another iterator:
```

Comment:
Its a good question -- [this says] that they are both interchangeable. I am just English :P